File: 27_caesars_cipher.js
// caesars cipher

function rot13(str){
    var solved = "";

    for(var i=0; i < str.length; i++){
        var asciiNum = str[i].charCodeAt();
        if(asciiNum >= 65 && asciiNum<=77)
        {
            solved += String.fromCharCode(asciiNum + 13);
        }
        else if(asciiNum >=78 && asciiNum <=90)
        {
            solved += String.fromCharCode(asciiNum - 13);
        }
        else
        {
            solved += str[i];
        }
    }
    return solved;
}

//[a,b,c,d,e,f,g,h,i,j,k,l,m,n,o,p,q,r,s,t,u,v,w,x,y,z]
console.log(rot13("A"));
console.log(rot13("SERR PBQR PNZC"))

File: 27_caesars_cipher.py
# caesars cipher

def rot13(string):
    solved = ''

    for i in range(len(string)):
        asciiNum = ord(string[i])
        if(asciiNum >= 65 and asciiNum <= 77):
            solved += chr(asciiNum + 13)
        elif(asciiNum >= 78 and asciiNum <= 90):
            solved += chr(asciiNum -13)
        else:
            solved += string[i]
    return solved


print(rot13('A'))
print(rot13('SERR PBQR PNZC'))